src/components/parameter/ParameterEdit.vue

```diff
@@ -124,70 +188,133 @@
 
             <template v-if="state.options.input === 'tickSlider'">
               <v-subheader class="paramLabel" v-text="label()" />
-              <v-tooltip left v-model="state.showResolutionWarning">
-                <template v-slot:activator="{ on, attrs }">
-                  <v-slider
-                    :max="state.options.ticks.length - 1"
-                    :thumb-color="state.color"
-                    :tick-labels="state.options.ticks"
-                    @change="paramChange"
+              <v-slider
+                :hide-details="state.message.length === 0"
+                :hint="state.message"
+                :max="state.options.ticks.length - 1"
+                :persistent-hint="state.message.length > 0"
+                :rules="rules"
+                :thumb-color="state.color"
+                :tick-labels="state.options.ticks"
+                :value="state.value"
+                @change="paramChange"
+                class="mb-1"
+                dense
+                height="40"
+                tick-size="4"
+                ticks="always"
+              >
+                <template #message>
+                  <div
+                    @click="closeMessage"
                     class="mb-1"
-                    dense
-                    height="40"
-                    hide-details
-                    ticks="always"
-                    tick-size="4"
-                    :value="state.value"
-                    v-bind="attrs"
-                    v-on="on"
-                  />
+                    style="cursor: pointer"
+                  >
+                    <v-divider />
+                    <v-row class="mx-0 py-1">
+                      <v-col class="text-center" cols="2">
```

Comment:
A colored background would help to assign the warning to the slider at a very first glance. Mapping multiple lines of elements (text, sliders, ...) to the correct paragraph is not an intuitive action anymore, if the number of lines increases.

```suggestion
                      <v-col
                        class="text-center"
                        cols="2"
                        style="background-color: #eeeeee"
                      >
```